tests/test_feeds.py

```diff
@@ -0,0 +1,270 @@
+import pytest
+from datetime import timedelta
+
+from django.http import Http404
+
+from nomination import feeds
+from . import factories
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestURLFeed:
+
+    def test_get_object(self, rf):
+        request = rf.get('/')
+        project = factories.ProjectFactory()
+        feed = feeds.url_feed()
+        obj = feed.get_object(request, project.project_slug)
+        assert obj == project
+
+    def test_get_object_raises_404(self, rf):
+        request = rf.get('/')
+        feed = feeds.url_feed()
+        with pytest.raises(Http404):
+            feed.get_object(request, '')
+
+    def test_title(self):
+        project = factories.ProjectFactory()
+        feed = feeds.url_feed()
+        title = feed.title(project)
+        assert title == 'Latest URLs for {}'.format(project.project_name)
```

Comment:
How would you feel about making this a little more explicit. Something like this maybe?

```python
project = factories.ProjectFactory(project_slug='test-project')
# ...
assert  'Latest URLs for test-project' == title
```